start.py

```diff
@@ -1167,7 +1170,27 @@ def _configure_for_streamsets_before_start(deployment, cluster_name):
             break
 
 
+def _configure_yarn(deployment, cluster, cluster_name):
+    logger.info('Configuring Yarn ...')
+    configs = {'container_executor_min_user_id': '150'}
```

Comment:
There's a config in this role config group called `container_executor_allowed_system_users`. Let's get the current value and just append `sdc` instead of changing this `min_user_id`